benchexec/tools/fizzer.py

```diff
@@ -70,8 +70,15 @@ def cmdline(self, executable, options, task, rlimits):
         data_model = get_data_model_from_task(task, {ILP32: ["--m32"], LP64: []})
         if data_model is None:
             data_model = []
+        time_options = []
+        if rlimits.cputime is not None:
+            time_options += [ "--max_seconds", str(int((865.0/900.0) * rlimits.cputime))]
+            time_options += [ "--optimizer_max_seconds", str(int((30.0/900.0) * rlimits.cputime))]
+        memory_options = []
+        if rlimits.memory is not None:
+            time_options += [ "--max_exec_megabytes", str(int((13.0/15.0) * (rlimits.memory / (1024.0 * 1024.0))))]
```

Comment:
There are some magic numbers here such as 865, 900, 30, etc. Where do these come from and what do they mean?
Is this how your tool is supposed to be executed in general, and will this remain true for future versions as well?
Note that this tool-info module is not specific to SV-COMP, people will likely also use it to benchmark Fizzer in other circumstances, so it should not hard-code something that is specific to SV-COMP. And it should not hard-code anything that you likely do not want for future versions.